test/aws/osml/image_processing/test_gdal_tile_factory.py

```diff
@@ -223,6 +223,31 @@ def test_create_map_tiles_for_image(self):
                 assert tile_dataset.RasterYSize == 256
                 assert tile_dataset.GetDriver().ShortName == GDALImageFormats.PNG
 
+    def test_create_map_tiles_for_color_image(self):
```

Comment:
Not to address now - but I have been adding comments to our tests to describe what they are doing in other packages. Just leaving a breadcrumb for this here.